crates/services/bls_aggregation/README.md

```diff
@@ -0,0 +1,4 @@
+# Bls Aggregation Service
+
+## Example
```

Comment:
add example?

Reply:
we don't have one . leaving it as a placeholder